### PR TITLE
Group resources by time

### PR DESCRIPTION
**Pull request by xepozz**
| Q             | A
| ------------- | ---
| Is bugfix?    | ❌
| New feature?  | ✔️
| Breaks BC?    | ❌
| Fixed issues  | 

Now debug files will group by timestamp
image

### CODE REVIEW

File: src/Storage/FileStorage.php [hunk 1]
```diff
@@ -62,16 +62,17 @@ public function getData(): array
 
     public function flush(): void
     {
+        $basePath = $this->path . '/' . date('Y-m-d H:i:s') . '/' . $this->idGenerator->getId() . '/';
```
Comment (rustamwin) [inline src/Storage/FileStorage.php:65]: I think, time format is not needed here
```suggestion
        $basePath = $this->path . '/' . date('Y-m-d') . '/' . $this->idGenerator->getId() . '/';
```

**Review by rustamwin — COMMENTED** [review]
I like the idea, but need to check that `gc` is working properly.

**Comment by rustamwin** [comment]
> I like the idea, but need to check that gc is working properly.

I checked, it works.

**Comment by samdark** [comment]
👍

**Pull request merged**